### DISCUSSION TITLE
Modpack as Dependency

### DISCUSSION THREAD

**Discussion by dguisti** [modrinth/.github#94] [Feature requests]
### Is your feature request related to a problem? Please describe.

When creating modpacks for a specific genre, I often want to append standard optimization (sodium, etc.) and QOL mods (masa's mods, Xaeros, etc.). Modpacks like Fabulously Optimized already do a good job of maintaining all of these, but it's it a lot of work to download the mods from the optimization modpack and then manually update the root modpack every time they have updates.

### Describe the solution you'd like

I would like it if a modpack (rather than just a mod) could be added as a dependency for a modpack. So then I could create a modpack and tell it to include Fabulously Optimized. That way, whenever FO gets updated my modpack will automatically pull the updated optimization mods without me having to import them myself.

### Describe alternatives you've considered

This feature may already exist, but I did not see anywhere in the documentation talking about it, so I assume it's not available yet. I've considered writing an addition for some launcher that can add support for that, but I don't think there's any way that could work without support for modpacks as dependencies being added to the official spec.

**Comment by nightpool** [comment]
agreed! this would also be helpful for maintaining two or three "versions" of a modpack depending on whether users want e.g. extra shaders, animations, etc.
